src/measure_theory/decomposition/radon_nikodym.lean

```diff
@@ -18,7 +18,7 @@ most notably conditional expectations and probability cumulative functions.
 
 ## Main results
 
-* `measure_theory.measureabsolutely_continuous_iff_with_density_radon_nikodym_derive_eq` :
+* `measure_theory.measure.absolutely_continuous_iff_with_density_radon_nikodym_derive_eq` :
```

Comment:
```suggestion
* `measure_theory.measure.absolutely_continuous_iff_with_density_radon_nikodym_deriv_eq` :
```
Can you also check whether the name is correct in the file?

Reply:
good catch!